openstack/grafana/templates/_deployment.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: extensions/v1beta1
```

Comment:
Note: this file begins with _ so is not deployed, Please ignore contents for now.